bucket/tesseract-languages.json

```
@@ -5,10 +5,12 @@
     "version": "4.0.0",
     "url": [
         "https://github.com/tesseract-ocr/tessdata_fast/archive/4.0.0.zip",
+        "https://github.com/tesseract-ocr/tessdata/raw/d87b3cbc75555bd3282e0cadab5e159e2d468396/equ.traineddata",
```

Use raw.githubusercontent.com

```suggestion
        "https://raw.githubusercontent.com/tesseract-ocr/tessdata/d87b3cbc75555bd3282e0cadab5e159e2d468396/equ.traineddata",
```